.github/CODEOWNERS

```diff
@@ -49,6 +49,8 @@
 
 /addons/account/tests/test_portal_attachment.py @odoo/rd-accounting @odoo/rd-images
 
+/addons/auth_*/ @odoo/rd-security
```

Comment:
This would be considered a "Generic fallback rules" to be put in first part of the file as this targets multiple module names.

Need to decide if it comes before or after some of the existing rules depending on the goal. Eg. it one of those `auth_` modules contain `ir_http.py` file, should it match for website, security, or both?

Reply:
That's why I introduced it there: less changes to reach the desired effect :p

But I guess it's not a big deal to also notify rd-security for all `ir_http` and `ir_qweb` files. We can try that, if rd-website cares about what's inside the auth modules. It's better than duplicating those rules to special-case them for `auth*`, I think.

Reply:
I think we need to update the structure of the codeowners file actually, I had a similar thought in another PR. (see https://github.com/odoo/odoo/pull/71589#discussion_r643788415)

We have a need to distinguish wildcard rules that are there for simplicity (to list many modules at once, which I think is actually a bad reason and module should probably be listed one by one instead).

And then we have wildcard rules that we want to enforce as absolute priority: security team for the auth modules, SM team for the mail templates, ir_http/ir_qweb for website (and if that was me I'd also add security team on those), etc.

So rules would be ordered as such:

- modules with wildcard for lazy people
- modules with full name
- wildcard for priority files/folders
- override of priority wildcard in some specific modules (eg. a team wants to receive notification too for wildcarded rules)

Reply:
Doesn't the matching stop at the first match?

Reply:
No it's the other way around, it's the last matching rule that is used.